eclass/git-r3.eclass

```diff
@@ -355,6 +362,8 @@ _git-r3_set_gitdir() {
 	# now replace all the slashes
 	repo_name=${repo_name//\//_}
 
+	EGIT_DIR_NAME+=("${repo_name}")
```

Comment:
Put full path so you wouldn't have to deal with different values of `EGIT3_STORE_DIR`, and so you wouldn't have to reinvent it for every VCS out there.

Reply:
The full path will break if the user moves DISTDIR after they have installed a git-r3-based package.

Reply:
There is no requirement that `EGIT3_STORE_DIR` is under `DISTDIR`. In fact, I use a completely separate top-level directory for all VCS caches.